components/Links/index.tsx

```diff
@@ -0,0 +1,76 @@
+"use client";
+
+import { motion } from "framer-motion";
+
+const UsefulLinks = () => {
+  const links = [
+    {
+      label: "CICA",
+      url: "https://sigarra.up.pt/feup/pt/web_base.gera_pagina?P_pagina=21101",
+    },
+    {
+      label: "Sasup",
+      url: "https://www.up.pt/portal/pt/sasup/",
+    },
+    {
+      label: "Serviços Académicos",
+      url: "https://sigarra.up.pt/feup/pt/uni_geral.unidade_view?pv_unidade=73",
+    },
+    {
+      label: "Uni – A FEUP no teu bolso",
+      url: "https://play.google.com/store/apps/details?id=pt.up.fe.ni.uni&hl=pt_PT",
+    },
+    {
+      label: "Time Table Selector (TTS)",
+      url: "https://tts.niaefeup.pt/planner",
+    },
+  ];
```

Comment:
Should this be a payload?
Also, I think is important to add a description

Reply:
Done :)